docs/Configuration/yaml-files.md

```diff
@@ -21,6 +21,8 @@ team_settings: # Only teams/team-name.yml
 
 Currently, managing labels and users is only supported using Fleet's UI or [API](https://fleetdm.com/docs/rest-api/rest-api) (YAML coming soon).
 
+You may also wish to create specialized API-Only users which may modify configurations through GitOps, but cannot access fleet through the UI. These specialized users can be created through `fleetctl user create` with the `--api-only` flag, and then assigned the `GitOps` role, and given global or team scope in the UI.
+
```

Comment:
@dantecatalfamo mind updating this to show an example of the full command to create an API-only user with the GitOps role? (Less work for the user, they don't have to look up how to assign a role, what options are required, etc.)

Reply:
I actually don't know if creating a gitops team user is possible with fleetctl, it seems really poorly documented

Reply:
@rachaelshaw It looks like users can only be assigned global roles using `fleetctl`, not team roles